Tests/fixtures/smartphone-17.yml

```diff
@@ -3632,15 +3632,14 @@
   user_agent: Mozilla/5.0 (Linux; Android 8.1.0; ARTEL U3_4G Build/NMF26F) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/76.0.3809.111 Mobile Safari/537.36 AlohaBrowser/2.17.3
   os:
     name: Android
-
     version: 8.1.0
     platform: ""
   client:
     type: browser
     name: Aloha Browser
     version: 2.17.3
-    engine: WebKit
-    engine_version: "537.36"
+    engine: Blink
+    engine_version: ""    
```

Comment:
end 4 spaces should be removed 'engine_version: ""
'

Reply:
Done